flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/decorators/ExternalServiceDecoratorTest.java

```diff
@@ -118,5 +119,17 @@ public void testSetServiceExposedType() throws IOException {
         assertEquals(
                 KubernetesConfigOptions.ServiceExposedType.ClusterIP.name(),
                 ((Service) servicesWithClusterIP.get(0)).getSpec().getType());
+
```

Comment:
Have such assert into a separate test looks better. And let's try to use the `assertThat` in the new introduced test.